pkg/repositories/transformers/workflow.go

```diff
@@ -53,6 +53,7 @@ func FromWorkflowModel(workflowModel models.Workflow) (admin.Workflow, error) {
 		Closure: &admin.WorkflowClosure{
 			CreatedAt: createdAt,
 		},
+		ShortDescription: workflowModel.ShortDescription,
```

Comment:
I think we're missing writing the short description to the workflow model in the first place.